### ISSUE TITLE
Review asserted children of 'anatomical junction'

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/uberon#1795]
Stemming from https://github.com/obophenotype/uberon/pull/1774#issuecomment-771704481, copying here as a quick reminder to take a look:

Looking at the asserted children of 'anatomical junction', 
- [x] I wonder if 'perineal body' is indeed a junction (def says it's located at a junction, which doesn't necessarily make it a junction?)
- [x] should 'angle of oral opening' be better axiomatised?
- [x] 'esophagogastric junction' is axiomatised differently from its siblings; 
- [x] and there may be a few more classes needing work? (I focused on the ones that don't have 'connects' axioms, but didn't look at them all.)

**Comment by paolaroncaglia** [comment]
@cmungall (and/or anyone else who'd be willing to double-check some human-centric anatomy terms)
I edited the placement of some former asserted children of 'anatomical junction' that didn't look like strict junctions to me. Tagging you as some of those terms were added by you. I went ahead and merged as I think those changes are fairly safe and I'm aware of the demands on your time. I'll leave this ticket open for some time in case you or others wish to comment on [these edits](https://github.com/obophenotype/uberon/commit/dfa7f05e3baf827a7f73ab9eb0dd2d9bdc99eaba), then I'll close. Thanks.
